exam/templates/exam_home.html

```diff
@@ -1,11 +1,23 @@
 {% extends 'base.html' %}
-
+{% load static %}
 {% block content %}
 <div class="content">
     <div class="card">
         <h5 class="card-title">
             Exam Home Page (under construction)
         </h5>
     </div>
+    <div class="card">
```

Comment:
Students Bulk upload is supposed to be in `students_home.html` or `students.html` (probably in `students_home.html`)(in the `students` section), and not in `exam` section